enterprise/models.py

```diff
@@ -1801,6 +1801,17 @@ def get_program_enrollment_url(self, program_uuid):
 
         return utils.update_query_parameters(url, {'catalog': self.uuid})
 
+    def save(self, *args, **kwargs):
+        """
+        Copies the ``content_filter`` of a related CatalogQuery into this
+        instance's ``content_filter``.
+        """
+        if self.enterprise_catalog_query and self.sync_enterprise_catalog_query:
+            content_filter_from_query = self.enterprise_catalog_query.content_filter
```

Comment:
Would there be any harm to setting self.content_filter = self.enterprise_catalog_query.content_filter and saving it every time even if they are already the same?

Reply:
Hmm, that's a good point.  Since we're in the `save()` method, here, I think the answer is "no", and it'll save us a comparison.